reco_utils/evaluation/spark_diversity_evaluation.py

```diff
@@ -358,17 +358,14 @@ def distributional_coverage(self):
         """
```

Comment:
Could you add a reference in the doc string as to where in the literature this definition of coverage can be found?